### PR TITLE
incomplete run details change

### PR DESCRIPTION
**Pull request by M-Hietala**
# Choose a PR Template

Switch to "Preview" on this description then select one of the choices below.

<a href="?expand=1&template=data_plane_template.md">Click here</a> to open a PR for a Data Plane API.

<a href="?expand=1&template=control_plane_template.md">Click here</a> to open a PR for a Control Plane (ARM) API.

### CODE REVIEW

File: specification/ai/Azure.AI.Projects/agents/runs/models.tsp [hunk 1]
```diff
@@ -146,13 +146,20 @@ model ThreadRun {
   parallelToolCalls?: boolean;
 }
 
+/**
+ * Details on why the run is incomplete. Will be `null` if the run is not incomplete.
+ */
+model IncompleteRunDetails {
+  @doc("he reason why the run is incomplete. This indicates which specific token limit was reached during the run.")
```
Comment (nick863) [inline specification/ai/Azure.AI.Projects/agents/runs/models.tsp:153]: typo: he -> The

**Review by nick863 — APPROVED** [review]

File: specification/ai/Azure.AI.Projects/agents/runs/models.tsp (hunk 1 above)
Comment (jhakulin) [inline specification/ai/Azure.AI.Projects/agents/runs/models.tsp:153]: he -> The

**Review by jhakulin — APPROVED** [review]

File: specification/ai/Azure.AI.Projects/agents/runs/models.tsp (hunk 1 above)
Comment (M-Hietala) [reply specification/ai/Azure.AI.Projects/agents/runs/models.tsp]: fixed

Comment (M-Hietala) [reply specification/ai/Azure.AI.Projects/agents/runs/models.tsp]: fixed

**Review by dargilco — APPROVED** [review]

**Pull request merged**